com.unity.multiplayer.mlapi/package.json

```diff
@@ -2,8 +2,8 @@
     "name": "com.unity.multiplayer.mlapi",
     "displayName": "MLAPI Networking Library",
     "version": "0.1.0",
-    "unity": "2019.4",
```

Comment:
This means the min for MLAPI becomes 2020.3?

Reply:
Correct :D